core/src/impl/Kokkos_Core.cpp

```diff
@@ -166,6 +166,8 @@ int get_device_count() {
 #elif defined(KOKKOS_ENABLE_OPENACC)
   return acc_get_num_devices(
       Kokkos::Experimental::Impl::OpenACC_Traits::dev_type);
+#elif defined(KOKKOS_ENABLE_OPENMPTARGET)
+  return Kokkos::Experimental::OpenMPTarget::detect_device_count();
```

Comment:
If this is the only place where this function is used, I would prefer just inlining it.

Reply:
Yes this is the only place that this function is called for now. I was just following the style of other backends.